module3/sql/db.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE users_db;
```

Comment:
Add migration logic to fill other hosts with this data on start.

Reply:
Done